crates/cargo-wdk/README.md

```diff
@@ -90,10 +90,10 @@ The recommended way to do this is to [enter an eWDK developer prompt](https://le
 
 ## Driver Package Signature Verification
 
-`signtool verify` requires the certificate to be installed as in the `Trusted Root Certification Authorities` for this verification to succeed. These tasks are not enabled by default as the default behavior of `WDR` is to sign with a generated test certificate. These test certificates are typically only installed into `Trusted Root Certification Authorities` on computers dedicated to testing drivers, and not personal development machines, given the security implications of installing your own root certificates.
-
-If you understand these implications, and have installed the test certificate, then you may validate the signatures as follows:
+Verification using the `signtool verify` command requires the certificate to be installed in the development system's `Trusted Root Certification Authorities`. Therefore, the verification tasks are not enabled by default. However, they can be enabled as follows: 
 
 ```pwsh
 cargo wdk build --verify-signature=true
-```
\ No newline at end of file
+```
+
+**NOTE**: The above command creates a **test certificate** in the `Trusted Root Certification Authorities` and uses the same certificate for the verification. The `build` command does not allow passing your own certificates at the moment.
```

Comment:
First part of the statement is not correct as the tool won't create certificate in `Trusted Root Certification Authorities` list, instead it expects it to be present there (installed manually by developer for now) so that the command with --`verify-signatures` would succeed.